src/lib/routing-table.js

```diff
@@ -102,4 +124,12 @@ class RoutingTable {
   }
 }
 
+function getPrefixLength (str1, str2) {
```

Comment:
Apparently its not just a convention anymore.. I'll update this function accordingly.